cmd/main.go

```diff
@@ -116,7 +115,7 @@ var (
 func init() { //nolint:gochecknoinits
 	utilruntime.Must(componentApi.AddToScheme(scheme))
 	utilruntime.Must(serviceApi.AddToScheme(scheme))
-	utilruntime.Must(infrastructurev1alpha1.AddToScheme(scheme))
+	utilruntime.Must(infrav1.AddToScheme(scheme))
```

Comment:
nit: we should probably register also the `v1alpha1` version to the scheme since both the versions are still served